crates/bevy_input/src/gamepad.rs

```diff
@@ -1240,6 +1240,83 @@ const ALL_AXIS_TYPES: [GamepadAxisType; 6] = [
     GamepadAxisType::RightZ,
 ];
 
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct GamepadRumbleIntensity {
```

Comment:
What do you think, does this struct make sense, or should I just keep two fields on the event instead?

Reply:
It's going to be a bit annoying to construct rumble requests

```rust
GamepadRumbleRequest {
  // ...
  intensity: GamepadRumbleIntensity { strong: 1.0, weak: 0.5 },
  // ...
}
```

But  I think it's fine as-is.

Reply:
I put in some convenience constants, so:

```rust
intensity: GamepadRumbleIntensity::WEAK_MAX
```

STRONG_MAX, and MAX. You could also define some yourself specific to your game.